knowledge-base/modify-html5-viewer-area-width-through-kendo-splitter.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Modify default Document Map and Parameter areas width in Html5 Report Viewer
+description: Resize the default areas sizes through the splitter
+type: how-to
+page_title: Change Side areas sizes in Html5 Viewer with the Kendo splitter
+slug: modify-html5-viewer-area-width-through-kendo-splitter
+position: 
+tags: 
+ticketid: 1398733
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>13.0.19.116+</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+	<tr>
+		<td>Report Viewer</td>
+		<td>HTML5 ASP.NET MVC</td>
+	</tr>
+</table>
+
+
+## Description
+Sometimes it may be necessary to increase/decrease the default areas' width of the _Html5 Report Viewer_ widgets.
```

Comment:
Use "...increase the width of the Document Map and Parameters Area inside report viewer..." instead of "default areas". Omit "HTML5" as the requirements above already specify that the article is about the HTML5 ASP.NET MVC wrapper.